src/sql/arrow_sql_gen/postgres.rs

```diff
@@ -484,6 +484,55 @@ pub fn rows_to_arrow(rows: &[Row]) -> Result<RecordBatch> {
                         None => builder.append_null(),
                     }
                 }
+                Type::TIMESTAMPTZ => {
```

Comment:
Let's add an integration test for this

Reply:
Postgres timestamptz -> arrow is covered as part of the round trip integration test as shown in the screenshot. Shall we create a new one-way test just for pg timestamptz row -> arrow, or use the existing integration test.
![image](https://github.com/user-attachments/assets/9cd26f83-3c2d-43b5-a966-f28c0177b9f4)

Reply:
Value checking addressed in the new commit